pkg/extensions/search/schema.graphql

```diff
@@ -67,10 +76,11 @@ type LayerInfo {
 }
 
 type Query {
-  CVEListForImage(image: String!) :CVEResultForImage 
-  ImageListForCVE(id: String!) :[ImgResultForCVE]
-  ImageListWithCVEFixed(id: String!, image: String!) :ImgResultForFixedCVE
-  ImageListForDigest(id: String!) :[ImgResultForDigest]
-  ImageListWithLatestTag:[ImageInfo]
-  ExpandedRepoInfo(repo: String!):RepoInfo
+     CVEListForImage(image: String!) :CVEResultForImage
+     ImageListForCVE(id: String!) :[ImageInfo]
+     TagListForCve(id: String!, image: String!, getFixed: Boolean) :[ImageInfo]
```

Comment:
`TagListForCve` is confusing, I don't know if I'm searching for images affected by a CVE or unaffected by a CVE.
Can we use `ImageListForFixedCVE`? Why the name change?